packages/flutter/lib/src/material/search.dart

```diff
@@ -108,6 +108,13 @@ abstract class SearchDelegate<T> {
   /// The current value of [query] can be used to determine what the user
   /// searched for.
   ///
+  /// Note that this method might be called multiple times just like any other
+  /// build function in Flutter. Please cache the search results if you don't
+  /// want to call search backend multiple times on the same query.
+  /// See:
```

Comment:
We usually don't include issue references like this in API doc comments.

Reply:
Done.